### PR TITLE
Issue 126: Added separate Profile page styled accordingly to prototype, mobile-friendly 

### LINKED ISSUE #126: Profile Page

### PR DESCRIPTION
**Pull request by Andrii-I**
closes #126

### CODE REVIEW

**Comment by Andrii-I** [comment]
General "sanity check" from 1 person should be enough

File: src/client/components/RegistrationModal.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ import { useHistory, useLocation } from 'react-router-dom';
 import Button from './Button';
 import Modal from './Modal';
 
+import '../style/LoginModal.css';
```
Comment (zzheng07) [inline src/client/components/RegistrationModal.js:7]: Normally, each component should have its own css file. If there are styles specific to `RegistrationModal`, I suggest we move them to a seperate css file. For those common styles (used by multiple components), we can move them `index.css` for easy maintenance.

File: .env.save [hunk 2]
```diff
@@ -0,0 +1,4 @@
+DB_HOST=database-1.cfwynfjcelo6.us-east-1.rds.amazonaws.com
```
Comment (zzheng07) [inline .env.save:1]: Extra file?

File: src/client/pages/Profile.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        password: 'test',
+        email: 'test'
+      }
+    };
+    fetch('http://localhost:5000/api/register', request);
+    console.log('registration successful');
+  };
+
+  return (
+    <body>
```
Comment (zzheng07) [inline src/client/pages/Profile.js:34]: Using `body` tag inside another `body` tag is not recommended. And the `main` tag here adds additional nesting level. Maybe we can use some simplification here.

File: src/client/pages/Profile.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+      <Navbar />
+      <div className="profile-card">
+        <main>
+          <div className="registration-modal-body">
```
Comment (zzheng07) [inline src/client/pages/Profile.js:38]: Pay attention to the className here. They are from a different component.

File: src/client/components/RegistrationModal.js (hunk 1 above)
Comment (Andrii-I) [reply src/client/components/RegistrationModal.js]: Created a separate RegistrationModal.css, cleaned up Login.css
Still, common/standard style across all pages is something that we should use.

File: .env.save (hunk 2 above)
Comment (Andrii-I) [reply .env.save]: Extra file deleted

File: src/client/pages/Profile.js (hunk 3 above)
Comment (Andrii-I) [reply src/client/pages/Profile.js]: simplified tag structure: removed unnecessary <body> amd <main> tags

File: src/client/pages/Profile.js (hunk 4 above)
Comment (Andrii-I) [reply src/client/pages/Profile.js]: Changed className to appropriate values

**Comment by Andrii-I** [comment]
All commits/changes folded into issue112/pull request https://github.com/allenzhai/Psychify/pull/162

**Pull request closed without merging**